extensions/amp-image-slider/0.1/amp-image-slider.css

```diff
@@ -30,19 +30,19 @@ amp-image-slider {
 }
 
 .i-amphtml-image-slider-left-mask {
-  position: relative;
+  position: absolute;
   width: 100%;
-  transform: translateX(-50%);
   height: 100%;
   overflow: hidden;
-  z-index: 1;
 }
 
 .i-amphtml-image-slider-right-mask {
-  position: absolute;
+  position: relative;
   width: 100%;
+  transform: translateX(50%);
   height: 100%;
   overflow: hidden;
+  z-index: 1;
 }
 
 amp-image-slider amp-img > img {
```

Comment:
As is, this can be overridden if the user specifies something for this CSS selector. I think this is reasonable behavior (to specify defaults that can be overridden), but just wanted to double check that this is intended. If you don't want this to be overridden, you may want to specify an internal class or to !important.

Reply:
I believe we probably should allow user to override